osquery/sql/virtual_table.cpp

```diff
@@ -28,6 +28,13 @@ FLAG(uint64,
      0,
      "Add an optional microsecond delay between table scans");
 
+FLAG(bool,
+     extensions_default_index,
+     true,
+     "Enable INDEX (and thereby constraints) on all extension table columns. "
```

Comment:
Let's keep this succinct and move the long-form documentation into `./docs/wiki/installation/cli-flags.md`.

Reply:
That's guaranteed to get out of date.  Perhaps we change the `--help` to truncate by default unless `--verbose` is also passed?  And then pull in the full form output into our cli_flags.md file somehow as part of the build process?

Reply:
> That's guaranteed to get out of date.

Agree, generally, but that's not by-design and on a per-flag basis the descriptions are rather stable.

> Perhaps we change the --help to truncate by default unless --verbose is also passed?

I am not sure how to do this right now.

I'd like to be consistent for now, thus my recommendation to but the long form documentation into the wiki. The documentation for this flag should not change over time, right?